### PR TITLE
Export function getPageViewId

### PR DESCRIPTION
**Pull request by bradfrizzell**
This needs to be exposed because it needs to be called separately from the class that was using it for building AmpContext objects

### CODE REVIEW

**Comment by bradfrizzell** [comment]
@lannka

File: src/service/document-info-impl.js [hunk 1]
```diff
@@ -92,6 +93,9 @@ export class DocInfo {
  * @param {!Window} win
  * @return {string}
  */
-function getPageViewId(win) {
-  return String(Math.floor(win.Math.random() * 10000));
+export function getPageViewId(win) {
+  if (!pageViewIdByWindow[win]) {
+    pageViewIdByWindow[win] = String(Math.floor(win.Math.random() * 10000));
```
Comment (jridgewell) [inline src/service/document-info-impl.js]: `win` is an object, so this evaluates to `pageViewIdByWindow["[object Window]"]` every time.

Comment (bradfrizzell) [inline src/service/document-info-impl.js]: that was dumb of me. changed it to store it on the window object itself

**Comment by lannka** [comment]
@bradfrizzell I thought we should get the pageViewId from the host page?

**Comment by lannka** [comment]
As discussed, this is moved into #6506

**Pull request closed without merging**